examples/main/README.md

```diff
@@ -208,6 +208,14 @@ Top-p sampling, also known as nucleus sampling, is another text generation metho
 
 Example usage: `--top-p 0.95`
 
+### Min P Sampling
+
+-   `--min-p N`: Sets a minimum base probability threshold for token selection (default: 0.0).
```

Comment:
```suggestion
-   `--min-p N`: Sets a minimum base probability threshold for token selection (default: 0.05).
```

Reply:
So you are approving of my decision to keeping this on by default? Or should it be standardized as off by default for now?

Reply:
@cebtenzzre not sure we can just change the default like that + does adding it on by default with top-k and top-p still enabled?

Reply:
> not sure we can just change the default like that

The default was already changed everywhere else, I'm just suggesting that we document it consistently. Unless we decide not to enable it by default - I don't have a strong opinion because I use other interfaces to llama.cpp when I am trying to achieve specific sampler settings.

Reply:
I see. I'd already noticed the inconsistency but was hesitant to push until I got the word on whether or not we want to pursue it being on by default

Reply:
Let's keep it on by default and see what is the feedback when it gets merged